src/tests/run_load_tests.sh

```diff
@@ -1,7 +1,5 @@
 #!/usr/bin/env bash
 
-docker build --cache-from smartsdk/quantumleap -t smartsdk/quantumleap ../../
```

Comment:
don't we need this line? I think w/o it, when you run the script you get the latest image in the docker registry which doesn't not necessarily reflects the latest code changes? for example, say I check out a branch someone is working on, then run this script. The tests will run with my local image, which won't have the changes in the branch?

Reply:
correct, but for load test i thought it was not needed. can be inserted back

Reply:
cool, we could put it back in after rebasing #347 on master? or is it easier to do in this PR?